tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/bytetype/BByteArrayValueNegativeTest.java

```diff
@@ -67,5 +66,8 @@ public void testBlobValueNegative() {
         BAssertUtil.validateError(result, index++, "missing plus token", 16, 23);
         BAssertUtil.validateError(result, index++, "invalid base64 content in byte array literal", 17, 23);
         BAssertUtil.validateError(result, index++, "invalid base64 content in byte array literal", 18, 23);
+        BAssertUtil.validateError(result, index++, "invalid operation: type 'byte' does not support indexing", 19, 18);
```

Comment:
Let's move this test to org.ballerinalang.test.types.typedesc.TypedescTests as discussed. Can you also add another test similar to

```ballerina
public function main() {
    typedesc t = int[]|string[];
}
```

Reply:
sure